services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/S3NativeClientConfiguration.java

```diff
@@ -67,6 +67,11 @@ public S3NativeClientConfiguration(Builder builder) {
         TlsContextOptions clientTlsContextOptions =
             TlsContextOptions.createDefaultClient()
                              .withCipherPreference(TlsCipherPreference.TLS_CIPHER_SYSTEM_DEFAULT);
+
+        if (builder.httpConfiguration != null
+            && builder.httpConfiguration.shouldTrustAllCertificates() != null) {
+            clientTlsContextOptions.withVerifyPeer(!builder.httpConfiguration.shouldTrustAllCertificates());
```

Comment:
Should we log a warning message here if it's enabled?
```
SSL Certificate verification is disabled. This is not a safe setting and should only be used for testing.
```

Reply:
I had following concern
-> User had disabled the SSL certificate validation for testing but user doesnot want anyone to know about it. If we are loggin it then we are making this info available to everyone who is accessing the logs.

Because of this this reason I was not sure if we should log this , what do you say about this scenario ?

Reply:
We log a warning message in other HTTP clients, so it should be fine :) 
https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java#L698-L701

Reply:
Thanks